app/modules/sightings/models.py

```diff
@@ -30,6 +30,9 @@ class Sighting(db.Model, FeatherModel):
 
     assets = db.relationship('SightingAssets')
 
+    # TODO Can a sighting have multiple job IDs outstanding at any one point or only one?
+    jobs = db.Column(db.JSON, nullable=True)
+
```

Comment:
Sage can support multiple jobs for the same set of annotations and doesn't care. You _could_ start a job for a sighting and then another when the first had not yet completed.

We have at times discussed preserving multiple done jobs for review- like one identification job against your data and another against all data that you could review in parallel, although this isn't on the table for MVP.

Reply:
OK I'm thinking this json is actually now a jobId & status list and should be in annotation of course